hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -1048,7 +1048,7 @@ public void testSetSnapshotSummary() throws Exception {
   @Test
   public void testNotExposeTableProperties() {
     Configuration conf = new Configuration();
-    conf.set("iceberg.hive.table-property-max-size", "0");
+    final long maxHiveTablePropertySize = 0;
     HiveTableOperations ops =
         new HiveTableOperations(conf, null, null, catalog.name(), DB_NAME, "tbl");
```

Comment:
Same here. `conf` and `ops` are no longer needed.

Reply:
Fixed.